boards/samd20-xpro/include/periph_conf.h

```diff
@@ -124,10 +124,8 @@ static const tc32_conf_t timer_config[] = {
     }
 };
 
-#define TIMER_0_MAX_VALUE   0xffff
-
 /* interrupt function name mapping */
-#define TIMER_0_ISR         isr_tc3
+#define TIMER_0_ISR         isr_tcc0
```

Comment:
Should be `isr_tc0`
This device doesn't have TCC

Reply:
Yea but there is no `isr_tc0` - it's in the same place as `isr_tcc0`

Reply:
That's troublesome as users might be confuse by this.

Reply:
Do you suggest something like

```patch
diff --git a/cpu/samd21/vectors/vectors_samd2x.c b/cpu/samd21/vectors/vectors_samd2x.c
index b85aec8b1f..e805d2d8b0 100644
--- a/cpu/samd21/vectors/vectors_samd2x.c
+++ b/cpu/samd21/vectors/vectors_samd2x.c
@@ -49,6 +49,9 @@ WEAK_DEFAULT void isr_tcc0(void);
 WEAK_DEFAULT void isr_tcc1(void);
 WEAK_DEFAULT void isr_tcc2(void);
 WEAK_DEFAULT void isr_tcc3(void);
+WEAK_DEFAULT void isr_tc0(void);
+WEAK_DEFAULT void isr_tc1(void);
+WEAK_DEFAULT void isr_tc2(void);
 WEAK_DEFAULT void isr_tc3(void);
 WEAK_DEFAULT void isr_tc4(void);
 WEAK_DEFAULT void isr_tc5(void);
@@ -82,9 +85,21 @@ ISR_VECTOR(1) const isr_t vector_cpu[CPU_IRQ_NUMOF] = {
     isr_sercom3,            /* 12 Serial Communication Interface 3 */
     isr_sercom4,            /* 13 Serial Communication Interface 4 */
     isr_sercom5,            /* 14 Serial Communication Interface 5 */
+#ifdef TCC0
     isr_tcc0,               /* 15 Timer Counter Control 0 */
+#else
+    isr_tc0,                /* 15 Basic Control 0 */
+#endif
+#ifdef TCC1
     isr_tcc1,               /* 16 Timer Counter Control 1 */
+#else
+    isr_tc1,                /* 16 Basic Control 1 */
+#endif
+#ifdef TCC2
     isr_tcc2,               /* 17 Timer Counter Control 2 */
+#else
+    isr_tc2,                /* 17 Basic Control 2 */
+#endif
     isr_tc3,                /* 18 Basic Timer Counter 0 */
     isr_tc4,                /* 19 Basic Timer Counter 1 */
     isr_tc5,                /* 20 Basic Timer Counter 2 */
```
?

Reply:
Unfortunately... Yes...

Reply:
Ok.